doc/ipython-notebooks/structure/multilabel_structured_prediction.ipynb

```diff
@@ -266,7 +266,7 @@
     "    \n",
     "    test_labels = create_labels(Y_test, n_classes)\n",
     "    \n",
-    "    out_labels = LabelsFactory.to_structured(machine.apply(feats_test))\n",
```

Comment:
does this work without further downstream changes?

Reply:
see [here](https://github.com/shogun-toolbox/shogun/blob/986f97ee9d25898b8a3f75fa9976cc55305b4fdc/examples/undocumented/python/structure_hierarchical_multilabel_classification.py#L124) this is a similar situation and the example compiles correctly

Reply:
i see so it is just passed on. Cool! :)